context.extendedinfo.dialog/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="context.extendedinfo.dialog" name="ExtendedInfo Script - Open dialog" version="1.0.1" provider-name="phil65">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.extendedinfo" version="4.5.0"/>
+  </requires>
+  <extension point="kodi.context.item" library="addon.py">
+    <item>
+      <label>32000</label>
+      <visible>StringCompare(ListItem.dbtype,movie) | StringCompare(ListItem.dbtype,tvshow) | StringCompare(ListItem.dbtype,season) | StringCompare(ListItem.dbtype,actor) | StringCompare(ListItem.dbtype,director)</visible>
+    </item>
```

Comment:
This syntax is deprecated fyi: http://kodi.wiki/view/Context_Item_Add-ons Since you're PRing >=jarvis only